source/_integrations/homekit.markdown

```diff
@@ -149,6 +151,56 @@ homekit:
                 required: false
                 type: string
                 default: '`switch`'
+              stream_address:
+                description: Only for `camera` entities. The source IP address to use when streaming to RTP clients. If your Home Assistant host has multiple interfaces, selecting a specific IP may be necessary.
+                required: false
+                type: string
+                default: local IP from Home Assistant Core
+              stream_source:
+                description: Only for `camera` entities. A URL, file or other valid FFmpeg input string to use as the stream source, rather than the default camera source. Required for camera entities that do not natively support streaming (mjpeg). If `-i` is not found in the stream source, it is prepended to construct the FFmpeg input.
```

Comment:
```suggestion
                description: Only for `camera` entities. A URL, file or other valid FFmpeg input string to use as the stream source, rather than the default camera source. Required for camera entities that do not natively support streaming (MJPEG). If `-i` is not found in the stream source, it is prepended to construct the FFmpeg input.
```